### PR TITLE
chore(landing page): improve

### PR DESCRIPTION
**Pull request by AugustinMauroy**
Added feat:

* Logo on landing page
* Try-it on landing page
* small change on playground

### CODE REVIEW

**Review by anonrig — APPROVED** [review]

File: app/playground/page.tsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import type { FC } from 'react';
+
+// @TODO: implement in this repo
+const PlaygroundPage: FC = () => (
+    <div className="flex justify-center p-4">
+        <iframe src="https://playground.ada-url.com" width="90%" height="500px" />
```
Comment (anonrig) [inline app/playground/page.tsx]: I just pushed this before reviewing your code. Can you remove it and update your branch?

File: components/logo-display.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+"use client";
+import Image from 'next/image'
+import { useTheme } from 'next-themes'
+import type { FC } from 'react';
+
+const LogoDisplay: FC = () => (
+    <div className='flex flex-col items-center justify-center'>
+        <Image
+          src={`/temp-logo-${useTheme().theme === 'light' ? 'black' : 'white'}.svg`}
```
Comment (anonrig) [inline components/logo-display.tsx]: Can you run `npm run lint-fix`?

File: components/main-nav.tsx [hunk 3]
```diff
@@ -34,7 +34,7 @@ export function MainNav() {
           Performance
         </Link>
         <Link
-          href='https://playground.ada-url.com'
+          href='/playground'
```
Comment (anonrig) [inline components/main-nav.tsx]: These changes are also unnecessary with my latest push.

**Comment by AugustinMauroy** [comment]
Don't miss to push real artwork

**Review by anonrig — APPROVED** [review]

**Pull request merged**